### PR TITLE
Fixed Phalcon Developer Tools manual

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

File: transifex/base-rst/en/reference/tools.rst [hunk 1]
```diff
@@ -32,18 +32,18 @@
 .. code-block:: sh
 
    $ phalcon commands
-   
-   Phalcon DevTools (1.2.3)
-   
+
+   Phalcon DevTools (2.0.8)
```
Comment (beliarh) [inline transifex/base-rst/en/reference/tools.rst]: Файлы в `transifex/` вроде бы генерируются скриптом `generate-trstrings.php` на основе английской документации. Если так, то их давно пора обновить.

Comment (sergeyklay) [inline transifex/base-rst/en/reference/tools.rst]: Да, их нужно обновить. В этот PR эта задача не входит

**Comment by sergeyklay** [comment]
@SidRoberts any comments? :)

**Comment by SidRoberts** [comment]
Transifex? Tbh, I don't fully understand it. :wink: Would GitHub only have English and the other languages be "compiled" from Transifex submissions?

**Comment by sergeyklay** [comment]
Frankly I don't fully figured out with Transifex. The answer lies somewhere here https://github.com/phalcon/docs/pull/527 :) In any case, Transifex is not related to this PR.

**Comment by sergeyklay** [comment]
@niden

**Comment by niden** [comment]
All,

We have decided to remove Transifex from the equation. It has caused a lot of confusion and we want to keep things simple for everyone.

I am going to post a blog post today to explain. 

In short, any translations should go to the relevant repositories/language files.

**Comment by xboston** [comment]
@niden :+1:  very good!

**Pull request merged**